content/developer/tutorials/git.rst

```diff
@@ -0,0 +1,61 @@
+=====================
+Collaborate using git
+=====================
+
+Introduction
+============
+
+*What are git and github, what are there for*
+
+Setup git and github
+====================
+
+*Basically a link to the "Environment setup" section of the contributing/develoment*
+
+There are a few things on that page that I don't like
+
+* the windows installation wizard is super long and complicated (14 setup screens), especially to newcomers, there is no guide in the git book nor on the official https://gitforwindows.org/ website to help newcomers install git, which are the two documents referenced by our tutorial at the moment
+* ssh keys are PITA on windows as there is no agent (you need to install pageant)
+* the http protocol can be used via GCM https://github.com/git-ecosystem/git-credential-manager which is very well integrated to github
+* the passage about the PATH is weird, it should be right from the start on linux/mac. On Windows they should open Git Bash instead
+	* I suggest to have it inside a warning box "in case git doesn't work, lookup this page <link>"
+* "install Odoo from the sources" we should invite all odoo employees to install it in a specific folder, so there is a "standard" place share among us
```

Comment:
I don't think that's useful, except maybe to reference that path later in the setup guide, but it's very niche. In my team we all installed the sources at different places and it has never been an issue.